provisioner/clusterpy.go

```diff
@@ -312,11 +313,8 @@ func (p *clusterpyProvisioner) provision(
 		if postOptions.APIServerURL != "" {
 			cluster.APIServerURL = postOptions.APIServerURL
 		}
-		if postOptions.AZInfo != nil {
-			azInfo = postOptions.AZInfo
-		}
-		for k, v := range postOptions.TemplateValues {
-			values[k] = v
```

Comment:
Is this the code cleanup mentioned in the PR description?

Reply:
Okay, I just reread (3) from the description and it turns out it is. Then I suppose it's fine to remove it and replace it like it's done here.